patches/@loki+browser+0.35.0.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/node_modules/@loki/browser/src/disable-pointer-events.js b/node_modules/@loki/browser/src/disable-pointer-events.js
```

Comment:
Patch loki, so we could click some elements in Loki
https://metaboat.slack.com/archives/C063Q3F1HPF/p1721322103499079?thread_ts=1721286682.029039&cid=C063Q3F1HPF